core/build.gradle.kts

```diff
@@ -59,44 +45,23 @@ kotlin {
 publishing {
     publications {
         create<MavenPublication>("coreJVM") {
```

Comment:
Gradle will automatically set the artifactId and version based on the subproject (also in the POM), so I've removed the explicit assignment. If you want to leave it in to be explicit, that's an option too.